src/controllers/action-queue/types/pushTypes.ts

```diff
@@ -34,7 +44,12 @@ export type TxConfirmTrigger = {
   readonly txid: string
 }
 
-export type PushTrigger = AddressBalanceTrigger | PriceChangeTrigger | PriceLevelTrigger | TxConfirmTrigger
+export type PushTrigger = AddressBalanceTrigger | AllTrigger | AnyTrigger | PriceChangeTrigger | PriceLevelTrigger | TxConfirmTrigger
+
+/**
+ * Records when a trigger took place.
+ */
+export type PushTriggerState = undefined | Date
```

Comment:
This type should be:

```ts
export type PushTriggerState =
  | undefined
  // For "any" and "all" triggers:
  | PushTriggerState[]
  // For normal triggers:
  | Date
```

Reply:
I fixed it in the first commit. I'm going to leave the last commit intact for reworking the files to be more exact with the push-server files